zerver/webhooks/flock/doc.md

```diff
@@ -0,0 +1,24 @@
+Get Zulip notifications from your Flock channels.
+
+1. {!create-stream.md!}
+
+1. {!create-bot-construct-url-indented.md!}
+
+1. Go to **Apps** in the bottom-right corner.
```

Comment:
If it is something clickable, make this `Click on **Apps** in the bottom-right corner.`